{"name": "登录用户名错误", "status": "passed", "attachments": [{"name": "log", "source": "34ecb03f-6fbe-44d4-a43a-4c7860409b30-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "testcase_name", "value": "'登录用户名错误'"}, {"name": "testcase", "value": "{'name': '登录用户名错误', 'action': '/keyword/actions/login.yml', 'params': {'username': 18837774444}, 'validate': [{'type': 'page_contains', 'expect': '帐号密码登录'}]}"}], "start": 1634738073189, "stop": 1634738079174, "uuid": "f96c0b1c-d247-449b-8a45-01c49f91fba7", "historyId": "6bed0cf793b6887218b0ac9671b333d5", "testCaseId": "7b3c8701b03d8fb259b18b98c43b083a", "fullName": "keyword.keyword_run#test_keyword", "labels": [{"name": "parentSuite", "value": "keyword"}, {"name": "suite", "value": "keyword_run"}, {"name": "host", "value": "LAPTOP-93HH27QA"}, {"name": "thread", "value": "40944-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "keyword.keyword_run"}]}